### ISSUE TITLE
Loading environment values from a yaml

### ISSUE THREAD

**Issue by luxus1337** [chiel/omni-backup#21]
It would be useful to not have to set environment variables manually.

**Comment by chiel** [comment]
Yeah I'd actually like to run the entire project inside docker containers so you can just `docker-compose up -v` to start everything up. For Google auth we should maybe have a "demo" omni or something.
